org-code-javabuilder/playground/src/main/java/org/code/playground/Playground.java

```diff
@@ -0,0 +1,23 @@
+package org.code.playground;
+
+import java.util.HashMap;
+import org.code.protocol.*;
+
+public class Playground {
```

Comment:
nit: I think in the updated version of this API, `Playground` is a static final class - ok with merging this since it'll be updated shortly after when you implement the no-op API but just a heads up

Reply:
👍 good to know, thanks! I'm having a bit of a hard time following the discussion on that PR, I see this comment:

https://github.com/code-dot-org/javabuilder/commit/63bc14a55dc97f7fc26837c3420a9b8d0324c40b#commitcomment-54117605

Can you help me understand why we'd want one vs. the other?

Reply:
Yep sure - it's just so that the actual student facing API for viewing and manipulating the playground (e.g. `addImage`, `getWidth`, etc) can be part of a non-static class (which is now `Board.java`), while `Playground` simply owns a singleton instance of `Board`. This way, students can use the API in a pseudo-static manner without needing to instantiate a new `Board` (via something like `Board myBoard = Playground.board`) but the API can remain non-static, which makes internal implementation easier as `Board` will have to manage a lot of internal state and instance variables.